### ISSUE TITLE
[dup-selected] The dup-selected command leaves all rows still selected in the resulting sheet

### ISSUE THREAD

**Issue by frosencrantz** [saulpw/visidata#2225]
**Small description**
The dup-selected command leaves all rows still selected in the resulting sheet

**Expected result**
The selection is clearedin the new sheet.  Having all the rows selected feels less useful than the old behavior, which was to clear the selection.

**Actual result with screenshot**

https://asciinema.org/a/ZYzictbqzPyY00miko6y1sDnY

**Steps to reproduce with sample data and a .vd**

`vd sample_data/benchmark.csv`

Make a selection, (e.g. comma `,` on "Robert Armstrong"). then hit `"`  `dup-selected`. the resulting sheet has all rows selected.  Old versions of VisiData would clear the selection.  This change could break existing scripting.  

**Additional context**
Please include the version of VisiData and Python.  Latest develop branch.  Python 3.9.2.

**Comment by saulpw** [comment]
Thanks for reporting this, @frosencrantz. This was due to the fix for #2190.  I've reverted that fix, and fixed it in a different way which should not break other things.  This regression warrants a 3.0.1 release.

**Comment by frosencrantz** [comment]
Thanks @saulpw!  That seems to work.  #2190 was my guess as the cause, but I had no verification.  I wish I reported the problem earlier.  I had noticed something did not feel right, but I was not sure until I looked a little more closely.

I think the problem might still exist when going from a FreqSheet, and the 'gEnter' command.  I'm not able to provide a screenshot currently.  I think if you go to a frequency sheet, select a couple rows, then hit `gEnter` you get a new sheet, with all rows selected.  Previously I think no rows  were selected.  I think this is even after the latest changes.
